dnsapi/dns_http_net.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
it's not bash.

please check this guide : https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide